recipes/hocort/meta.yaml

```diff
@@ -45,7 +45,6 @@ test:
 about:
   home: https://github.com/ignasrum/hocort
   license: MIT
-  license_file: LICENSE
```

Comment:
No, please leave this in here. Conda will get the file named LICENSE from your tarball

Reply:
Sorry, I thought it referred to the LICENSE file that I removed.